cmd/meroxa/global/client.go

```diff
@@ -172,9 +173,31 @@ func NewClient() (meroxa.Client, error) {
 		}
 	}
 	options = append(options, meroxa.WithAccountUUID(Config.GetString(UserAccountUUID)))
+	options = append(options, meroxa.WithHeader("Meroxa-CLI", "1"))
+	options = append(options, meroxa.WithHeader("Meroxa-CLI-Version", Version))
+	if goVersion := getGoVersion(); goVersion != "" {
+		options = append(options, meroxa.WithHeader("Meroxa-CLI-Go-Version", goVersion))
```

Comment:
`Meroxa-CLI-Go-Version` -> `Meroxa-CLI-App-Version`

Reply:
all version is somewhere else

Reply:
this is the version of go that people are using to run the meroxa CLI.  it's not a turbine value. we could potentially use this value to determine if there's a minimum version of go that we need to check for before running any commands, etc.

Reply:
I see. Going to add that in the CLI PR as well.